android/.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,29 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Why do we need another .idea folder in the android folder?

Reply:
That way you can open the android project in Android Studio/Intellij and debug it from there.

Reply:
I also like this approach